src/actions/color/rgb.ts

```diff
@@ -0,0 +1,35 @@
+import Action from "../Action";
+import {IAction} from "../../interfaces/IAction";
+import Param from "../../parameters/Param";
+
+/**
+ * @implements IAction
+ */
+class Rgb extends Action implements IAction {
```

Comment:
This could in theory be a single class called "Color" right?
And based on the constructor values, create different params?

What do you think?